tests/run_travis.sh

```diff
@@ -24,8 +24,7 @@ if [[ $SHOTGUN_COMPILE_ONLY -eq 1 ]]; then
     exit 0
 fi
 
-
-if [[ $TRAVIS -eq true ]]; then
+if [[ $TRAVIS = true ]]; then
```

Comment:
Apparently this evaluates to true even when running the shell script on my computer. My shell-fu is really weak... 👎